docs/builders/vite.md

```diff
@@ -96,41 +96,27 @@ If you need, you can also configure Storybook's Vite builder using TypeScript. R
 
 ## Troubleshooting
 
-### Prebundle errors
+### Working directory not being detected
 
-Vite automatically restarts and begins the prebundling process if it detects a new dependency. This pattern breaks with Storybook and throws confusing error messages. If you see the following message in your terminal:
```

Comment:
@IanVS thanks for clearing this up, appreciate it, looking at the builder's Readme we need to also update it as well to avoid some confusion. I'll follow up with you on this.

Reply:
Yep, there are a few things I need to clean up there.  👍